teamscale-client/src/main/java/com/teamscale/client/ITeamscaleService.java

```diff
@@ -81,33 +81,32 @@
 	);
 
 	/** Retrieve clustered impacted tests based on the given available tests and baseline timestamp. */
-	@PUT("api/projects/{projectName}/impacted-tests")
+	@PUT("api/v6.5.2/projects/{projectName}/impacted-tests?prioritization-strategy=CHEAP_ADDITIONAL_COVERAGE_PER_TIME")
 	Call<List<PrioritizableTestCluster>> getImpactedTests(
 			@Path("projectName") String projectName,
-			@Query("baseline") Long baseline,
+			@Query("baseline") String baseline,
 			@Query("end") CommitDescriptor end,
-			@Query("partitions") String partition,
+			@Query("partitions") List<String> partitions,
```

Comment:
Did you change the corresponding Teamscale service query option, or did it already support multiple partitions?

Reply:
It already did. We just didn't expose it here yet